src/app/home/directives/home-series.component.ts

```diff
@@ -9,6 +9,9 @@ import { StoryModel } from '../../shared';
   styleUrls: ['home-series.component.css'],
   template: `
     <header *ngIf="noseries">
+      <section [publishHoverTip]="'These are your stories not associated with any Series.'" class="hover-tip">
```

Comment:
Cool, thanks! Docs for HostBinding weren't great -- went with `@HostBinding('attr.class') class = 'hover-tip';`. That ok?